app/src/main/java/com/e/salesapp/CategoriesListFragment.java

```diff
@@ -12,25 +12,24 @@
 
 public class CategoriesListFragment extends Fragment {
 
-    OnCategorySelected listener;
+    OnCategorySelected mCategoryListener;
 
     CategoriesListFragment(OnCategorySelected categorySelected)
     {
-        this.listener = categorySelected;
+        this.mCategoryListener = categorySelected;
```

Comment:
what does mean mCategoryListener?, all names that you use inside a class should named as properties of the class, please be aware with this deatail